etc/tempesta_fw.conf

```diff
@@ -1071,7 +1075,31 @@
 #   }
 #
 # Default:
-#   All limits are disabled (the values are set to zero/false/empty).
+#   frang_limits {
+#       ip_block off;
+#       request_rate 0;
+#       request_burst 0;
+#       tcp_connection_rate 0;
+#       tcp_connection_burst 0;
+#       concurrent_tcp_connections 0;
+#       tls_connection_rate 0;
+#       tls_connection_burst 0;
+#       tls_incomplete_connection_rate 0;
+#       client_header_timeout 0;
+#       client_body_timeout 0;
+#       http_header_chunk_cnt 0;
+#       http_body_chunk_cnt 0;
+#       http_uri_len 0;
+#       http_body_len 1073741824;
+#       http_hdr_len 0;
+#       http_header_cnt 0;
```

Comment:
50 is default value for this setting. Please also check all other settings

Reply:
Fixed